app/src/main/res/values/strings.xml

```diff
@@ -104,6 +104,8 @@
     <string name="settings_key_display_barcode_max_brightness" translatable="false">pref_display_card_max_brightness</string>
     <string name="settings_lock_barcode_orientation">Lock barcode orientation</string>
     <string name="settings_key_lock_barcode_orientation" translatable="false">pref_lock_barcode_orientation</string>
+    <string name="settings_keep_screen_on">Keep screen on</string>
```

Comment:
Maybe "Keep screen on when viewing loyalty card"? Just so people don't get confused why it may still turn off on other screens?